rpmlb/cli.py

```diff
@@ -126,3 +135,15 @@ def run(recipe_file, collection_id, **option_dict):
     builder.run(work, **option_dict)
 
     log.info('Success!')
+
+
+def _adjust_option_dict(option_dict):
+    if not option_dict:
+        return
+    download = option_dict.get('download', None)
+    # When specifying download type: fedpkg or rhpkg,
+    # users must want to use the command in Builder.
```

Comment:
That is not true. What about rebuilding from Fedora to (EP)EL/CentOS? In that case you might want a `fedpkg` download, but `rhpkg` build.

Better solution would be setting the `pkg_cmd` default to `None` and do something like this:

```python
# If the user did not request a pkg command,
# choose one based on relevant download type.

# user chose the command explicitly
if option_dict['pkg_cmd'] is not None:
    return

# no explicit command, but relevant builder was selected
if download in ('fedpkg', 'rhpkg'):
    option_dict['pkg_cmd'] = download
    return

# no pkg_cmd or relevant builder – set default value
option_dict['pkg_cmd'] = 'fedpkg'
```

At that point, the functions starts to look as a prime candidate for `pkg_cmd` callback, although that might be more complicated to use.